modules/external-api/src/main/java/org/opencastproject/external/endpoint/GroupsEndpoint.java

```diff
@@ -93,7 +93,7 @@
 @Path("/")
 @Produces({ ApiMediaType.JSON, ApiMediaType.VERSION_1_0_0, ApiMediaType.VERSION_1_1_0, ApiMediaType.VERSION_1_2_0,
             ApiMediaType.VERSION_1_3_0, ApiMediaType.VERSION_1_4_0, ApiMediaType.VERSION_1_5_0,
-            ApiMediaType.VERSION_1_6_0, ApiMediaType.VERSION_1_7_0 })
+            ApiMediaType.VERSION_1_6_0, ApiMediaType.VERSION_1_7_0, ApiMediaType.VERSION_1_8_0 })
```

Comment:
I think the API version needs to be changed.

Reply:
Just to be sure, the issue here is that the API version should already be 1_8_0 and needs to be bumped to 1_9_0 everywhere, right?

Reply:
Yes, I think so. Else it would add a feature to a released API version. @lkiesow, correct me if I am wrong.